beacon-chain/p2p/monitoring.go

```diff
@@ -11,6 +11,10 @@ var (
 		Help: "The number of peers in a given state.",
 	},
 		[]string{"state"})
+	repeatPeerConnections = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "p2p_repeat_attempts",
+		Help: "The number of repeat attempts the connection handler is triggered for a peer.",
```

Comment:
```suggestion
		Help: "The number of repeated attempts to connect to peers.",
```

Reply:
Dont agree with the change, its not repeated connection attempts, its the connection handler getting triggered multiple times, the distinction is important here. A peer can only be connected once to another peer.